.gitignore

```diff
@@ -1 +1,2 @@
 .idea
+cli.iml
```

Comment:
Probably need more entries in here.  This is what we have for depshield:
.classpath
.idea
.project
.settings/
.sonatype
target/
*.iml
*.eml
.DS_Store
*.pyc

Reply:
imho there's no need to add ignores until they become relevant but whatever the team decides upon